src/Raven.Server/Documents/AI/AiAssistant/Handlers/Processors/AiAssistantHandlerProcessorBase.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using JetBrains.Annotations;
+using Raven.Server.Commercial;
+using Raven.Server.Documents.Handlers.Processors;
+using Raven.Server.Web;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Documents.AI.AiAssistant.Handlers.Processors;
+
+internal abstract class AiAssistantHandlerProcessorBase : AbstractHandlerProcessor<RequestHandler>
+{
+    private readonly License _license;
+    private readonly string _certificateThumbprint;
+    
+    private const string LicensePropertyName = "License";
+    private const string CertificateThumbprintPropertyName = "CertificateThumbprint";
+
+    protected AiAssistantHandlerProcessorBase([NotNull] RequestHandler requestHandler) : base(requestHandler)
+    {
+
+        if (ServerStore.Configuration.Ai.DisableAiAssistant)
+            throw new InvalidOperationException("AI Assistant is disabled by the administrator.");
```

Comment:
Here we need to explain what is the configuration value and how to re-enable it.